### PR TITLE
Add cardano-wallet-js

### PR DESCRIPTION
**Pull request by leobel**
---

## Showcase addition

* Name: *cardano-wallet-js*
* Description: *A javascript/typescript SDK for Cardano Wallet with a extra functionalities. You can use it as a client for the official cardano-wallet and also to create Native Tokens and NFTs.* 
* Link:   https://github.com/tango-crypto/cardano-wallet-js
* Source: https://github.com/tango-crypto/cardano-wallet-js
* Tags:
  * `opensource`
  * `metadata`
  * `wallet`

 

 

---

### CODE REVIEW

File: docs/get-started/cardano-wallet-js.md [hunk 1]
```diff
@@ -0,0 +1,805 @@
...
+managing wallets if it's not running locally.
+
+# Requirements
+Before start using the library you will need a `cardano-wallet` server running. If you have docker available you can just
+download the `docker-composer.yml` they provide and start it using `docker-compose`:
+
+    wget https://raw.githubusercontent.com/input-output-hk/cardano-wallet/master/docker-compose.yml
+    NETWORK=testnet docker-compose up
+
+> **NOTE:** You can find more information about different options to start the cardano-wallet server [here](https://github.com/input-output-hk/cardano-wallet)
```
Comment (rdlrt) [inline docs/get-started/cardano-wallet-js.md:65]: Thanks for your PR.

Could you adapt the page to the [styling guide](https://developers.cardano.org/docs/portal-style-guide), as things like admonitions can look quite different than `>` marker.
(PS: not adding comments to each instance below for same)

File: docs/get-started/cardano-wallet-js.md [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+ - listing wallets
+
+Our project aims to provide an easy to use Javascript SDK for programmers, instead of
+exposing the raw REST structure to you. 
+
+Finally, it helps you to build desktop wallet clients - like [Daedalus](https://daedaluswallet.io/) - with
+embedded cardano-wallet binaries, so you don't necessarily have to 
+connect to a remote cardano-wallet server.
+
+## Warning
```
Comment (rdlrt) [inline docs/get-started/cardano-wallet-js.md:52]: Can this be changed to important admonition , and moved above Introduction?

(PS: Will leave it to you if you want to include, but the use case for cardano-wallet are mostly custodial services and full node wallet management - specifying few clients can become slightly ambiguous)

**Review by rdlrt — CHANGES_REQUESTED** [review]

File: docs/get-started/cardano-wallet-js.md (hunk 1 above)
Comment (leobel) [reply docs/get-started/cardano-wallet-js.md]: Hi @rdlrt, thanks for you suggestions, I did the changes you mention.

**Comment by leobel** [comment]
Hi, I'd like to know if we as the PR creators needs to do something else or just wait for more feedbacks. The PR has been quiet for a long time :(

**Comment by katomm** [comment]
> Hi, I'd like to know if we as the PR creators needs to do something else or just wait for more feedbacks. The PR has been quiet for a long time :(

No, all is well from your side, thanks.

**Review by katomm — APPROVED** [review]
Looks good, thank you @leobel

**Review by rdlrt — APPROVED** [review]

**Pull request merged**